include/lsst/meas/base/ApertureFlux.h

```diff
@@ -254,13 +251,16 @@ class ApertureFluxAlgorithm : public SimpleAlgorithm {
 struct ApertureFluxResult : public FluxResult {
 
     /// Return the flag value associated with the given bit
-    bool getFlag(ApertureFluxAlgorithm::FlagBits bit) const { return _flags[bit]; }
+    bool getFlag(unsigned int index) const { return _flags[index]; }
+
+    /// Return the flag value associated with the given flag name
+    bool getFlag(std::string name) const { return _flags[ApertureFluxAlgorithm::getFlagDefinitions().getDefinition(name).number]; }
```

Comment:
This line is too long.  There are also many other lines below that are beyond the 110 char limit.